config.php

```diff
@@ -67,8 +67,8 @@
 $iconv_input_encoding = 'UTF-8';
 
 // date() format for file modification date
-// Doc - https://www.php.net/manual/en/function.date.php
-$datetime_format = 'd.m.y H:i';
+// Doc - https://www.php.net/manual/en/datetime.format.php
+$datetime_format = 'd.m.y H:i:s';
```

Comment:
added _second_ to file modification date format

Reply:
my preference for _modification date_
but i can remove it with `git rebase -i` , if you don't prefer.